src/devhub/index.html

```diff
@@ -49,15 +54,15 @@ <h3>Links</h3>
         <h3>Main branch fuzzes</h3>
         <table>
           <tr>
-            <td class="ok" style="padding-right: 16px;">ok</td>
+            <td class="ok" style="padding-right: 16px;" title="This many fuzzes didn't have any failure.">ok</td>
             <td><span id="fuzz-ok">N/A</span></td>
           </tr>
           <tr>
-            <td class="fail" style="padding-right: 16px;">fail</td>
+            <td class="fail" style="padding-right: 16px;" title="This many fuzzes had a failure.">fail</td>
             <td><span id="fuzz-fail">N/A</span></td>
           </tr>
           <tr>
-            <td class="canary" style="padding-right: 16px;">canary</td>
+            <td class="canary" style="padding-right: 16px;" title="The canary fuzzer found this many failures.">canary</td>
```

Comment:
Can we explain the concept of "canary" here? i.e. What should the viewer expect to be "good" here? And how should they interpret this if it's bad?